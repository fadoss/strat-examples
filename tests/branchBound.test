params/branchBound/bb-traveler

srew initial(node(city(0), 0), generateFixData(4)) using solve .

load bb-traveler-list
srew initial(node(city(0), 0), generateFixData(3)) using solve .

load bb-traveler-index
srew initial(node(city(0), 0), generateFixData(5)) using solve .
