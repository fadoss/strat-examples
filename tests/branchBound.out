==========================================
srewrite in MAIN : initial(node(city(0), 0), generateFixData(4)) using solve .

Solution 1
rewrites: 1791
result BBState: solution(node(city(0) city(4) city(3) city(1) city(2) city(0),
    20), 20)

No more solutions.
rewrites: 2285
==========================================
srewrite in MAIN : initial(node(city(0), 0), generateFixData(3)) using solve .

Solution 1
rewrites: 1112
result BBState: solution(node(city(0) city(3) city(1) city(2) city(0), 15), 15)

No more solutions.
rewrites: 1236
==========================================
srewrite in MAIN : initial(node(city(0), 0), generateFixData(5)) using solve .

Solution 1
rewrites: 15472
result BBState: solution(node(city(0) city(4) city(3) city(1) city(2) city(5)
    city(0), 24), 24)

No more solutions.
rewrites: 16959
Bye.
